### PR TITLE
[patch] cycle keys, tidy pipeline

### PR DESCRIPTION
**Pull request by camjc**
Nothing runs except on master, so will need to merge and check afterward.

### CODE REVIEW

File: codeship-services.yml [hunk 1]
```diff
@@ -2,9 +2,6 @@ build-service:
   build:
     image: eslint-config
     dockerfile: Dockerfile
-    encrypted_args_file: codeship-args.encrypted
```
Comment (camjc) [inline codeship-services.yml:5]: Nothing private to install, so no need for this.

File: codeship-steps.yml [hunk 2]
```diff
@@ -1,24 +1,4 @@
...
-- name: Confirm ssh connection to server, authenticating with generated public ssh key
-  service: build-service
-  tag: master
-  command: /bin/bash -c "ssh -o StrictHostKeyChecking=no -Tvv git@github.com 2>&1 | grep 'successfully authenticated'"
-
-- name: Release the app
-  service: build-service
-  tag: master
-  command: npx -p @ahmdigital/devops release
+  command: /bin/bash -c "echo //registry.npmjs.org/:_authToken=$NPM_TOKEN_AHM > ~/.npmrc && npx -p @ahmdigital/devops authenticate_and_release && rm -f ~/.npmrc"
```
Comment (simonhoye) [inline codeship-steps.yml:4]: Whats going to happen here as we don't have an NPM token? Should we have a release script as well?

Comment (camjc) [reply codeship-steps.yml:4]: We do have a `write` token for `steps`, just not a read one for `dockerfile`. :)

File: codeship-env.encrypted [hunk 3]
```diff
@@ -1,2 +1,2 @@
 codeship:v2
```
Comment (camjc) [inline codeship-env.encrypted:1]: PRIVATE_SSH_KEY
NPM_TOKEN_AHM
GITHUB_OAUTH_TOKEN

File: codeship-steps.yml (hunk 2 above)
Comment (simonhoye) [reply codeship-steps.yml:4]: oh, of course... forget me

**Review by simonhoye — APPROVED** [review]
LGTM

**Pull request merged**